values/gatekeeper-operator/constraint-templates/k8sallowedrepos.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sallowedrepos
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sAllowedRepos
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          properties:
+            repos:
+              type: array
+              items:
+                type: string
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      libs:
+        - |-
+          {{- readFile "../../policies/lib/core.rego" | nindent 10 }}
```

Comment:
Above, for conftest, you seem to load many more files, such as helpers. Are those not needed here as well?

Reply:
all of these files have been removed now as we generate them on the fly from the prepare hook, calling Konstraint to build them in the artifacts folder.